keps/sig-network/0752-endpointslices/README.md

```diff
@@ -209,21 +211,14 @@ type Endpoint struct {
     // endpoint.
     // +optional
     TargetRef *v1.ObjectReference `json:"targetRef,omitempty" protobuf:"bytes,4,opt,name=targetRef"`
-    // topology contains arbitrary topology information associated with the
-    // endpoint. These key/value pairs must conform with the label format.
-    // https://kubernetes.io/docs/concepts/overview/working-with-objects/labels
-    // Topology may include a maximum of 16 key/value pairs. For endpoints
-    // backed by Kubernetes Pods, This may include, but is not limited to the
-    // following well known keys:
-    // * kubernetes.io/hostname: the value indicates the hostname of the node
-    //   where the endpoint is located. This should match the corresponding
-    //   node label.
-    // * topology.kubernetes.io/zone: the value indicates the zone where the
-    //   endpoint is located. This should match the corresponding node label.
-    // * topology.kubernetes.io/region: the value indicates the region where the
-    //   endpoint is located. This should match the corresponding node label.
+    // nodeName represents the name of the Node hosting this endpoint. This can
+    // be used to determine endpoints local to a Node. This field can be enabled
+	  // with the EndpointSliceNodeName feature gate.
+	  // +optional
+	  NodeName *string `json:"nodeName,omitempty" protobuf:"bytes,6,opt,name=nodeName"`
```

Comment:
Nit: indentation looks like it got messed up here.

Reply:
Thanks for catching that! Looks like tabs got added instead of spaces.